symengine/lib/symengine_wrapper.pyx

```diff
@@ -778,10 +790,38 @@ cdef class Basic(object):
         return False
 
     @property
+    def is_integer(self):
+        return False
+
+    @property
+    def is_finite(self):
+        return None
+
+    @property
     def is_Derivative(self):
         return False
 
     @property
+    def is_AlgebraicNumber(self):
+        return False
```

Comment:
`AlgebraicNumber` is a class in itself in `SymPy`, with no child classes. Only a small fraction of code uses this flag, and the passed object is never an `AlgebraicNumber`. Hence, I didn't thought it'd be useful in implementing the class.